Dockerfile

```diff
@@ -1,13 +1,15 @@
-FROM okteto/node:16 as builder
+FROM okteto/node:18 as builder
 
 WORKDIR /app/builder
 
-ENV NODE_ENV=development
 ENV PLAYWRIGHT_SKIP_BROWSER_DOWNLOAD=1
+ARG HUSKY=0
+ENV HUSKY=0
 
 COPY . /app/builder/
 
-RUN npm install
+RUN npm ci --ignore-scripts
+ENV NODE_ENV=production
```

Comment:
set this AFTER install so dev dependencies are installed. because we need to build everything in the docker image